### DISCUSSION TITLE
Will the integration method remove the differences between cancer and ctrl samples?

### DISCUSSION THREAD

**Discussion by hyjforesight** [satijalab/seurat#5086] [Q&A] [answered]
Hello all,
I'm wondering how to determine the cancer-specific or cancer-determining genes by comparing cancer dataset with ctrl dataset.
I'm using the Findermarker() function to get the DEG list by comparing the cancer dataset with ctrl. Then do GSEA.
Here comes the question.
If I merge these 2 datasets and do Findermarker(), the batch effect is not removed, which will result in false-positive DEGs.
If I do integration of these 2 datasets first to remove batch effects, because the integration method is to map 2 datasets based on their similarity, will this method erase the intrinsic differences between cancer dataset and ctrl? In another word, the DEGs will be undermined after integration.
Thanks!
Best
YJ

**Comment by timoast** [comment, marked as answer]
We recommend running differential expression using the uncorrected data (see https://github.com/satijalab/seurat/discussions/4000). The integration workflow allows you to identify what groups of cells are similar across the datasets, and so can guide you as to which groups of cells to compare in a test, but the integrated data should not be used in the test itself.

**Reply by hyjforesight** [reply]
Hello timoast,
Thanks for the answers.
I'm still wondering whether Findmarkers() is still capable of determining the DEGs between clusters in integrated data by setting the default assay to RNA?
For example, we integrate tumor data with wild-type data and find a distinct cluster which is specific for tumor after integration. Can we do DefaultAssay(integreated.data) <- "RNA" first and use FindAllMarkers() for this distinct cluster because the uncorrected data is still stored under the “assays", right?
Thanks!
Best,
YJ

**Reply by timoast** [reply]
Yes
